src/components/Login/Login.js

```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import { loginUser } from "../../auth/auth-actions";
+import { useAuthDispatch } from "../../auth/auth-context";
+import "./login.css";
+
+export const Login = () => {
+  const [username, setUsername] = useState();
+  const [password, setPassword] = useState();
+  const [message, setMessage] = useState("");
+
+  const dispatch = useAuthDispatch();
+
+  const loginClickHandler = async () => {
+    let payload = { username, password };
+    try {
+      let response = await loginUser(dispatch, payload);
+      console.log(response.data.success);
+      if (response.data.success) {
+        setMessage("Could not Login, try again");
+      } else setMessage("Happy Shopping!");
```

Comment:
Didn't get here. If success then message showing "Could not login"...Why?

Reply:
Was testing something and forgot to change that, my bad